README.md

```diff
@@ -47,8 +47,10 @@ $validationData = new ValidationData();
 $validationData->setIssuer('https://appleid.apple.com');
 $validationData->setAudience('com.azimo');
 
-$appleJwtFetchingService = new Auth\Service\AppleJwtFetchingService(
-            new Auth\Jwt\JwtParser(new \Lcobucci\JWT\Token\Parser(new \Lcobucci\JWT\Encoding\JoseEncoder())),
+//No use statements ?????
```

Comment:
When I usually see the code snippets there are no use statements. I have no problem with adding them here. Please feel free to add use statements :)